src/Yolov5Net.Scorer/YoloScorer.cs

```diff
@@ -305,26 +238,17 @@ private List<YoloPrediction> Supress(List<YoloPrediction> items)
         /// <summary>
         /// Runs object detection.
         /// </summary>
-        public List<YoloPrediction> Predict(Image image)
-        {
-            return Supress(ParseOutput(Inference(image), image));
-        }
+        public List<YoloPrediction> Predict(Image<Rgba32> image) => Suppress(ParseOutput(Inference(image), (image.Width, image.Height)));
```

Comment:
There is a bug that causes the detected rectangles to based on the size of the resized image and not the size of the original.

Because image is passed as a reference any changes will modify the original image object. This will case (image.Width, image.Height) to be the resized width and height.

The fix for this is to change Inference(image) to Inference(image.Clone())

Reply:
Yes, it is. I tried to resize the output to fit the origin size:
```cs
var (w, h) = (image.Width, image.Height);
...
image.Mutate(a => a.Resize(w, h));
```       
It works but the image was distorted. Your solution avoids this problem, Thanks!

Reply:
No problem, glad I could help.